rareapi/fixtures/category.json

```diff
@@ -0,0 +1,38 @@
+[
+
+    {
+        "model": "rare.category",
```

Comment:
I think these need to follow the file structure, so "model": "rareapi.category"

Reply:
this has been udpated!